vms/shift/templates/shift/edit_hours.html

```diff
@@ -1,6 +1,22 @@
 {% extends "vms/base.html" %}
 
 {% block content %}
+<script type="text/javascript">
```

Comment:
@tapasweni-pathak yes, there is a commom JS file. But we don't need the timepicker JS code everywhere. I can create a new JS file only for the timepicker and use it. What is the best way to import in Django a JS file to the .html file (edit_hours.html)?